api/handler.go

```diff
@@ -43,3 +43,21 @@ func sendError(w http.ResponseWriter, r *http.Request, code int, err error) {
 		RequestId: requestIDFromContext(r.Context()),
 	})
 }
+
+func sendTaskError(w http.ResponseWriter, r *http.Request, code int, err error) {
```

Comment:
I think you can simplify `sendTaskError` and `sendTaskDeleteError` to `sendError` by just using the oapigen.ErrorResponse object. Anything consuming the API should still be able to unmarshal as expected, since the ErrorResponse object will match the embedded error response.

Reply:
Oh, that makes sense! I didn't really love these methods anyway because I figured the amount would just grow as we added more response types. Will make the necessary changes!